### PR TITLE
Use baseline_clients_daily instead of ping and first_seen for fenix_android_clients

### PR DESCRIPTION
**Pull request by ANich**
Follow-up from https://github.com/mozilla/bigquery-etl/pull/3863#discussion_r1207004233 

Updates `firefox_android_clients` to use `baseline_clients_daily` instead of `baseline_clients_first_seen` and the baseline ping.

### CODE REVIEW

**Review by fbertsch — APPROVED** [review]
LGTM

File: sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/init.sql [hunk 1]
```diff
@@ -1,24 +1,44 @@
...
     normalized_os_version AS os_version,
     app_display_version AS app_version,
-    locale
+    locale,
+    is_new_profile,
   FROM
-    `moz-fx-data-shared-prod.fenix.baseline_clients_first_seen`
+    `moz-fx-data-shared-prod.org_mozilla_firefox.baseline_clients_daily`
   WHERE
     submission_date >= '2019-01-01'
```
Comment (lucia-vargas-a) [inline sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/init.sql:21]: `fenix.firefox_android_clients` has data for channel release from `2020-01-21`.

However I am noticing that `moz-fx-data-shared-prod.fenix.baseline_clients_first_seen` has data from `2019-06-21` for channel release, while `moz-fx-data-shared-prod.org_mozilla_firefox.baseline_clients_daily` has data from `2020-01-21`.

The older data comes from `moz-fx-data-shared-prod.org_mozilla_**fenix**.baseline_clients_first_seen` where the min. submission_date is `2019-06-21`.

I'm highlighting it to make sure we are happy to have data from this date or if we might need the older data in the future we'd need to join both sources. What do you think?

**Review by lucia-vargas-a — APPROVED** [review]
LGTM.
Only a comment to check if data from '2020-01-21' is ok.

File: sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/init.sql (hunk 1 above)
Comment (lucia-vargas-a) [inline sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/init.sql:21]: For clarity in the query
```suggestion
    submission_date >= '2020-01-21'
```

**Pull request merged**